.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "env"
+  ]
+}
```

Comment:
Doesn't your editor add newlines for you?

Reply:
Not on dot files. Perhaps I could configure that though.

Reply:
Fixed

Reply:
My diff still shows no newline, same thing in the `.eslintrc`, `.npmignore`, `Dockerfile` and `docker-compose.yml` files. You sure your setting is working :stuck_out_tongue: